src/main/kotlin/com/ampnet/mailservice/service/impl/UserMailServiceImpl.kt

```diff
@@ -185,10 +227,11 @@ class UserMailServiceImpl(
             logger.debug("There is no attachment ${project.tosUrl}")
             null
         }
-        successfullyInvestedMail
-            .setTemplateData(
-                SuccessfullyInvestedTemplateData(project, request.amount.toLong(), user.coop.name, termsOfService)
-            )
+        val mail = if (termsOfService == null) successfullyInvestedWithoutTosMail
+        else successfullyInvestedMail
```

Comment:
```suggestion
        val mail = if (termsOfService == null) successfullyInvestedWithoutTosMail else successfullyInvestedMail
```
What is the value for intellij preferences -> editor -> code style -> kotlin -> wrapping -> hard wrap at? It should be 120.